src/client/eppo-client.ts

```diff
@@ -748,6 +757,17 @@ export default class EppoClient {
     return result;
   }
 
+  private getFlagEvaluationDetailsBuilder(flagKey: string): FlagEvaluationDetailsBuilder {
```

Comment:
I went ahead and renamed since I don't feel too strongly, but the pattern in this codebase seems to have a `get` prefix for simple creator/producer methods, which is what this is. Other examples are `getAssignmentDetail`, `getConfigDetails`, etc. There probably should be a code style guide or ESLint rule codified somewhere if we're trying to align on a standard.